tests/features/collection/og.feature

```diff
@@ -75,10 +81,11 @@ Feature: Organic Groups integration
     And I should see the "Join this collection" button
     And the "Überwaldean Land Eels" collection should have 0 members
 
-    When I go to the homepage of the "Folk Dance and Song Society" collection
+    When I am logged in as "Kathie Cumberwatch"
+    And I go to the homepage of the "Folk Dance and Song Society" collection
```

Comment:
I updated the comment for leaving the collection that applies when your membership is approved. 
As for the test, the whole feature is to join and leave collections. Requesting a membership is part of the flow depending of collection values. The functionality to cancel their pending memberships was not built deliberately. I thought that this would allow email spamming if users cancel and re request memberships. + there are no specs for that at the moment.